### ISSUE TITLE
Documentation on where the data is backed up & how often

### ISSUE THREAD

**Issue by TetianaKhomenko** [testomatio/docs#49]
original thread https://testomatio.slack.com/archives/C064BCVR3BJ/p1699974960694819

**Our respond:**

We try to maintain multi-backup policy both on system level and on user level.

We have incremental backup of whole database with 7 days retention. So we can restore data on any point in time from the latest 7 days. Few weeks ago we faced an incident where client accidentally deleted their tests and we could restore it from the backup.

Also we have Pulse, a versioned storage, where all revisions of tests and suites are stored up to 90 days. So deleted tests and suites can also be restore by user from the interface. We have run tests where over 1100 tests can be restored after deletion from the interface.

However, that's not all we have. We plan to add two new backup strategies, both on system and user level
To distribute risks of storing database backup, we will connect a 3rd party service https://snapshooter.com/ to maintain backups for a period of time longer than 7 days. The data will be stored in encoded mode

To provide users more control over their data we will add automatic project backup to S3 storage in XLS format. So this will allow storing and restoring tests and suites for period longer than 90days.

To sum up, here is our backup policy:

**System level:**

- Database backup (7 days)
- Alternative database backup (over 7 days) (PLANNED)

**User level:**

- Pulse (90 days)
- Automatic project backup in XLS (over 90 days) (PLANNED)
